### ISSUE TITLE
[backlog] Tests for install process in docker compose context

### ISSUE THREAD

**Issue by anthony-sarkis** [diffgram/diffgram#257]
In the context of this regression
https://github.com/diffgram/diffgram/issues/255

It would be great if there was an automatic test for this

**Comment by PJEstrada** [comment]
Yes I agree this is a good idea too. I think with a regex with could parse the docker compose files to see if there is the same amount of variables as in settings.py to to this test. This could also be a good idea for the helm chart value.yaml
